pyproject.toml

```diff
@@ -179,7 +180,13 @@ packages = ["frequenz.dispatch"]
 strict = true
 
 [[tool.mypy.overrides]]
-module = ["mkdocs_macros.*", "async_solipsism", "async_solipsism.*"]
+module = [
+  "mkdocs_macros.*",
+  "async_solipsism",
+  "async_solipsism.*",
+  "grpc.aio",
+  "grpc.aio.*",
```

Comment:
Just upgrading the SDK made it start complaining about this? Weird.

Reply:
Yes, I ran nox without any changes and all the checks passed.
Then I updated channels, and again all checks passed.
After updating the SDK dependency to:

```diff
diff --git a/pyproject.toml b/pyproject.toml
index 76ee55c..47227e5 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -39,7 +39,7 @@ dependencies = [
   # Make sure to update the version for cross-referencing also in the
   # mkdocs.yml file when changing the version here (look for the config key
   # plugins.mkdocstrings.handlers.python.import)
-  "frequenz-sdk == 1.0.0rc601",
+  "frequenz-sdk == 1.0.0-rc801",
   "frequenz-channels >= 1.1.0, < 2.0.0",
   "frequenz-client-dispatch >= 0.5.0, < 0.6.0",
 ]
 ```

And I got the following errors in mypy:
```sh
nox > Running session mypy
nox > Creating virtual environment (virtualenv) using python in .nox/mypy
nox > python -m pip install -e '.[dev-mypy]'
nox > mypy 
src/frequenz/dispatch/actor.py:10: error: Skipping analyzing "grpc.aio": module is installed, but missing library stubs or py.typed marker  [import-untyped]
src/frequenz/dispatch/actor.py:10: error: Skipping analyzing "grpc": module is installed, but missing library stubs or py.typed marker  [import-untyped]
src/frequenz/dispatch/_dispatcher.py:9: error: Skipping analyzing "grpc.aio": module is installed, but missing library stubs or py.typed marker  [import-untyped]
src/frequenz/dispatch/_dispatcher.py:9: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
src/frequenz/dispatch/_dispatcher.py:9: error: Skipping analyzing "grpc": module is installed, but missing library stubs or py.typed marker  [import-untyped]
Found 4 errors in 2 files (checked 6 source files)
nox > Command mypy  failed with exit code 1
nox > Session mypy failed.
```